db-client-java/src/main/java/com/eventstore/dbclient/EventStoreDBProjectionManagementClient.java

```diff
@@ -11,44 +11,316 @@ private EventStoreDBProjectionManagementClient(EventStoreDBClientSettings settin
+        return this.create(projectionName, query, CreateProjectionOptions.get());
+    }
+
+    /**
+     * Creates a new projection in the stopped state. Enable needs to be called separately to start the projection.
+     * @param projectionName Name of the projection.
+     * @param query The JavaScript projection.
+     * @param options Additional options.
+     */
+    public CompletableFuture create(final String projectionName, final String query, CreateProjectionOptions options) {
         if (options == null)
-            options = CreateContinuousProjectionOptions.get();
+            options = CreateProjectionOptions.get();
 
         if (!options.hasUserCredentials())
             options.authenticated(this.credentials);
 
-        return CreateProjection.forContinuous(this.client, projectionName, query, options).execute();
+        return new CreateProjection(this.client, projectionName, query, options).execute();
     }
 
-    public CompletableFuture createOneTime(final String projectionName, final String query) {
-        return this.createOneTime(projectionName, query, CreateOneTimeProjectionOptions.get());
+    /**
+     * Enables the projection.
+     * @param projectionName Name of the projection.
+     */
+    public CompletableFuture enable(final String projectionName) {
+        return this.enable(projectionName, EnableProjectionOptions.get());
     }
 
-    public CompletableFuture createOneTime(final String projectionName, final String query, CreateOneTimeProjectionOptions options) {
-        if (options == null)
-            options = CreateOneTimeProjectionOptions.get();
+    /**
+     * Enables the projection.
+     * @param projectionName Name of the projection.
+     * @param options Additional options.
+     */
+    public CompletableFuture enable(final String projectionName, EnableProjectionOptions options) {
+        return new EnableProjection(this.client, projectionName, options).execute();
+    }
 
-        if (!options.hasUserCredentials())
-            options.authenticated(this.credentials);
+    /**
+     * Deletes the projection.
+     * @param projectionName Name of the projection.
+     */
+    public CompletableFuture delete(final String projectionName) {
+        return this.delete(projectionName, DeleteProjectionOptions.get());
+    }
+
+    /**
+     * Deletes the projection.
+     * @param projectionName Name of the projection.
+     * @param deleteEmittedStreams Whether the streams that have been emitted by the projection should be deleted.
+     *                             This only has an effect if trackEmittedStreams was enabled
+     * @param deleteStateStream Whether the projection's state stream should also be deleted.
+     * @param deleteCheckpointStream Whether the projection's checkpoint stream should also be deleted.
+     */
+    public CompletableFuture delete(final String projectionName, final boolean deleteEmittedStreams,

```

Comment:
**SUGGESTION:** I think that we may remove this method. We have an options class to provide such params.